addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -98,18 +99,33 @@ def _get_bom(self, bom_id=False, product_id=False, line_qty=False, line_id=False
             'bom_prod_name': product.display_name,
             'currency': self.env.user.company_id.currency_id,
             'product': product,
+            'code': bom.code,
             'price': product.uom_id._compute_price(product.standard_price, bom.product_uom_id) * bom_quantity,
             'total': sum([op['total'] for op in operations]),
             'level': level or 0,
             'operations': operations,
             'operations_cost': sum([op['total'] for op in operations]),
+            'attachments': self.env['mrp.document'].search(['|',
+            '&', ('res_model', '=', 'product.product'), ('res_id', '=', product.id), '&', ('res_model', '=', 'product.template'), ('res_id', '=', product.product_tmpl_id.id)]),
```

Comment:
wrong indent

Reply:
you're changing this line at rev 6c8654fbd9d2aa8d430f20bc03db39200c7e110f

please fix these commits